src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write code here
+const allEmployers = [...document.querySelectorAll('li')];
+
+const getSalary = (el, attr) => {
+  return el.getAttribute(attr)
+    .slice(1)
+    .split(',')
+    .join('');
+};
+
+const attributeToGet = 'data-salary';
+
+const sortBySalaries = allEmployers.sort((a, b) => {
```

Comment:
what is it? if it is employers than it should be:
```suggestion
const sortedEmployers = allEmployers.sort((a, b) => {
```

Reply:
also please don't use naming like`a, b`